docs/variants/tuxedo_ibs15/post_install.md

```diff
@@ -0,0 +1,24 @@
+# Post-installation setup
```

Comment:
Is this specific to Tuxedo?

Reply:
It's common for Clevo laptops with the touchpad disable hotkey.

Reply:
We should think how to organize this content in long run, since we now agree to take documentation debt, which we will have to pay with margin later.